netman/adapters/switches/brocade.py

```diff
@@ -308,7 +308,7 @@ def get_vlan_interfaces(self, vlan_number):
             if regex.match("(Untagged|Statically tagged) Ports\s+: (.*)$", line):
                 for real_name in _to_real_names(parse_if_ranges(regex[1])):
                     interfaces.append(real_name)
-        return interfaces
+        return sorted(interfaces)
```

Comment:
Why do we need to sort here?

Will it output ["ethernet 1", "ethernet 2", "ethernet 3"... "ethernet 10", "ethernet 11"] or ["ethernet 1", "ethernet 10", "ethernet 11", "ethernet 2", "ethernet 3" ...]